src/base/preferences.cpp

```diff
@@ -1483,3 +1483,25 @@ void Preferences::apply()
     if (SettingsStorage::instance()->save())
         emit changed();
 }
+
+// Preview Modal Dialog
```

Comment:
use of preference class is deprecated in this case, follow `m_storeDialogSize` in torrentcreatordlg.cpp for a modern approach.

Reply:
I'm going to check it

Reply:
Yeah understand, I refactor it today

Reply:
>Refactored to use CachedSettingValue<> class

OK, the implementation looks good to me 👍, now you should get rid of the newly added functions here since they won't be used.

Reply:
Ohh, I wanted, but forgot, my fault :/, thx for review